openreview/conference/templates/areachairWebfield.js

```diff
@@ -891,6 +907,39 @@ var registerEventHandlers = function() {
     })
     return false;
   });
+
+  $('#group-container').on('click', '#select-all-papers', function(e) {
```

Comment:
For checkboxes, it's better to listen to the 'change' event, because users can check and uncheck with the keyboard as well.